src/Node/Blockcore.Node/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Threading.Tasks;
+using Blockcore.Builder;
+using Blockcore.Configuration;
+using Blockcore.Features.SignalR;
+using Blockcore.Features.SignalR.Broadcasters;
+using Blockcore.Features.SignalR.Events;
+using Blockcore.Utilities;
+
+namespace Blockcore.Node
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            try
+            {
+                string chain = args
+                   .DefaultIfEmpty("--chain=BTC")
+                   .Where(arg => arg.StartsWith("--chain", ignoreCase: true, CultureInfo.InvariantCulture))
+                   .Select(arg => arg.Replace("--chain=", string.Empty, ignoreCase: true, CultureInfo.InvariantCulture))
+                   .FirstOrDefault();
+
+                NodeSettings nodeSettings = NetworkSelector.Create(chain, args);
+                IFullNodeBuilder nodeBuilder = NodeBuilder.Create(chain, nodeSettings);
+
+                if (nodeSettings.EnableSignalR)
```

Comment:
argh this stuff should be hidden in the feature (well that's for another PR)

Reply:
The SignalR parts? Yes it should.